src/Microsoft.PowerShell.Commands.Utility/Microsoft.PowerShell.Commands.Utility.csproj

```diff
@@ -35,6 +35,7 @@
     <Compile Remove="commands\utility\ShowCommand\ShowCommandParameterType.cs" />
     <Compile Remove="commands\utility\ShowCommand\ShowCommandProxy.cs" />
     <Compile Remove="commands\utility\update-list.cs" />
+    <Compile Remove="commands\utility\Unblockfile.cs" Condition=" '$(Configuration)' == 'Linux' " />
```

Comment:
That's what I had originally.  Did a search and I don't see other cases of conditional compilation, so I think it's more consistent to use a whole file `#ifdef`.  @iSazonov ok?